### ISSUE TITLE
Stuck on connecting when using reverse proxy

### ISSUE THREAD

**Issue by ghaisasadvait** [boypt/simple-torrent#94]
When I access it directly from the local IP, it load perfectly.
I am using a NGINX Proxy Manager as a reverse proxy and whenever I am connecting using that, the page is stuck on connecting.
I tried entering the following variable lines in the **NGINX Proxy Manager's 'advanced' tab** as mentioned in the official documentation:    
proxy_set_header Connection '';  
proxy_http_version 1.1;

It's still not working.
I also tried using entering the following line :  proxy_pass http://127.0.0.1:3000/sync; mentioned in #5 

Still its not working. How should I solve this issue?

**Comment by boypt** [comment]
how about the config in wiki: https://github.com/boypt/simple-torrent/wiki/ReverseProxy

**Comment by ghaisasadvait** [comment]
I tried that already as mentioned above. However, this time, in the NGINX proxy Manager, I tried it by adding a custom location with path as 'sync' and the including the 2 variables(proxy_set_header Connection ''; AND proxy_http_version 1.1;

Below is the Image of the configuration.

![image](https://user-images.githubusercontent.com/11556546/99766829-e9bb0980-2b27-11eb-92c0-0d2126f84f87.png)

Thank your for the reply!
